sdlf-utils/glue-jobs-deployer/README.md

```diff
@@ -0,0 +1,78 @@
+# Motivation
+
+To deploy glue jobs as the process step in a stage in the SDLF is one of the most common tasks you have to accomplish once the framework is deployed and transformations based on glue jobs are added to be used as part of a pipeline. 
+
+# Solution
+
+This artifact is a solution, implemented as a folder structure, including a deploy.sh main script, to be able to automatically deploy several glue jobs, based on the concepts of pipeline and dataset from the SDLF.
+
+The artifact pretends easily create as much glue jobs as necessary trough a single shared Yaml CloudFormation template, for different “types” of processes organized in folders to be named as the related pipelines/datasets deployed in the framework.
+
+Using this structure, you will have a unique script (Python/Scala) per pipeline, and several Json parameter files as descriptors to manage the specific behavior of the script per dataset.
+
+The proposed **deploy.sh** script can be included as part of a code pipeline, to automatically create/update the new or changed dataset parameters files, from the last commit in the repository, so not all glue jobs are re-created each time you deploy. 
+
+![Architecture](Images/glue_deployer_4_sdlf_fs.png) 
+
+
+# Requirements
+
+* SDLF deployed in the target account
```

Comment:
Should we add that `jq` is required as pre-installed software on the machine since you use it in the deploy script?

Reply:
You right, thank you! added.